infra/filter.lua

```diff
@@ -16,6 +16,7 @@ function LoadMeta(meta)
   meta.rel = config.rel
   meta.base = config.base
   meta.draft = config.draft
+  meta.colorlinks = true
```

Comment:
What does this line do?

Reply:
Make links blue in the PDF. Weirdly, this is setting a pandoc setting, which then causes our template to generate different LaTeX with different settings. It would be cleaner to make this change entirely in LaTeX but right now the template is a bit of a mess and I didn't want to fight it too much.